server/models/Profile.js

```diff
@@ -0,0 +1,46 @@
+const mongoose = require("mongoose");
+
+const profileSchema = new mongoose.Schema(
+    {
+        firstname: {
+            type: String,
+            required: true,
+        },
+        lastname: {
+            type: String,
+            required: true,
+        },
+        gender: {
+            type: String,
+            required: true,
+        },
+        birthDate: {
+            type: Date,
+            required: true,
+        },
+        email: {
+            type: String,
+            required: true,
+            unique: true,
+        },
```

Comment:
This is already stored in the `User` model ([file here](https://github.com/seblaifa3315/fun-kickoff-project/blob/feature_profile_model/server/models/User.js)).

And in case you wanted to use it as a link to the user account, it's still better to it via `ref`s using `_id`s 👍🏼